iceoryx_posh/source/roudi/memory/posix_shm_memory_provider.cpp

```diff
@@ -78,7 +76,7 @@ cxx::expected<void*, MemoryProviderError> PosixShmMemoryProvider::createMemory(c
             .hasErrors())
     {
         LogFatal() << "Could not set signal handler for SIGBUS!";
-        std::terminate();
+        errorHandler(Error::kROUDI_MEMORY__COULD_NOT_REGISTER_SIGBUS, nullptr, ErrorLevel::FATAL);
```

Comment:
Please add a return after the `errorHandler`.

Reply:
Done